powerhome/playbook
# [PLAY-422] Passphrase kit dependency and documentation

#### Screens

<img width="1721" alt="Screen Shot 2022-11-04 at 2 13 44 PM" src="https://user-images.githubusercontent.com/8194056/200035863-96bbe3c9-7fe0-4130-9c3b-5cd91204cf25.png">

#### Breaking Changes

Yes, removed the zxcvbn library from playbook and added it only to the playbook-website.

Props related to the `useZxcvbn` and `useHaveIBeenPwned` custom hook were removed: `minLength`, `common`, `averageThreshold`, `strongThreshold`, `onStrengthChange` and `checkPwned`.

Instead of adding a new prop for strenght checking, I added an example showing how to do that with zxcvbn using the passphrase kit.

***Deprecation*** of the rails version of the kit. 

#### Runway Ticket URL

[[PLAY-422]](https://nitro.powerhrg.com/runway/backlog_items/PLAY-422) 

#### How to test this

Open milano build and check if the kit renders correctly. 
Check for any console errors.
Alpha testing on Nitro to check if components using the `Passphrase` kit are still working.

#### Checklist:
